opflexagent/config.py

```diff
@@ -30,6 +30,16 @@
                 default=['*'],
                 help=_("List of the physical networks managed by this agent. "
                        "Use * for binding any opflex network to this agent")),
+    cfg.ListOpt('firewall_map',
```

Comment:
That's a good point (no sense in re-inventing the wheel ;-)!  I'll see what code can be leveraged there. The main thing I need is the ability to select an implementation based on a strategy, where the strategy method returns a key that selects the strategy. I think they use some of the stevedore libs there, so I'll look if I can simplify the code some by leveraging that.